api/src/main/java/org/openmrs/api/CohortService.java

```diff
@@ -102,13 +102,6 @@
 	@Authorized({ PrivilegeConstants.GET_PATIENT_COHORTS })
 	public Cohort getCohortByName(String name) throws APIException;
 	
-	/**
-	 * @deprecated use {@link #getCohortByName(String)}
-	 */
```

Comment:
We do not deprecate and remove at the same time. After deprecation, we still leave it in for a number of versions to come until when we eventually remove it.

Reply:
@dkayiwa Fixed